scripts/bug_retriever.py

```diff
@@ -59,9 +59,16 @@ def retrieve_bugs(self):
         bugzilla.download_bugs(timespan_ids + labelled_bug_ids)
 
         # Try to re-download inconsistent bugs, up to three times.
+        previously_inconsistent_bugs = []
         for i in range(3):
-            bugs = bugzilla.get_bugs()
-            bug_ids = set(bug_snapshot.get_inconsistencies(bugs))
+            if i == 0:
```

Comment:
Add a comment before the if saying: `We look for inconsistencies in all bugs first, then, on following passes, we only look for inconsistencies in bugs that were found to be inconsistent in the first pass`.